### DISCUSSION TITLE
axe-core update

### DISCUSSION THREAD

**Discussion by Robanna777** [harlan-zw/unlighthouse#57] [Ideas]
Can you confirm the axe-core version? I’m not getting errors from unlighthouse that I am getting in Lighthouse 10.1.1, which I believe is built off axe-core 4.6. One, in particular, is aria-required-children which was implemented in 4.4.

Do you have any plans to update to the current stable axe-core?

Thanks,

**Comment by Robanna777** [comment]
Nevermind. I see now that it is using 9.6.8, which is axe-core 4.4. Something must have happened in a later axe release that triggers that error. I don’t know if it’s a bug or if the rule set has changed.

Thanks for your work on this project. Your Rock!

**Comment by harlan-zw** [comment]
Thanks :) Glad you could figure it out.

To give you a bit more context, I'm working on bumping Lighthouse to v10 but there are a few breaking changes that I need to sort out. Will be available soon so you might see this error pop up in newer versions.
